site/content/en/docs/concepts/topology_aware_scheduling.md

```diff
@@ -98,14 +98,35 @@ to a TAS ResourceFlavor.
 
 {{< include "examples/tas/sample-job-preferred.yaml" "yaml" >}}
 
+### ClusterAutoscaler support
+
+TAS integrates with the [Kubernetes ClusterAutoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler)
+through the [Provisioning AdmissionCheck](/docs/admission-check-controllers/provisioning/).
+
+When a workload is assigned to the TAS Resource Flavor with Provisioning
+AdmissionCheck, then its admission flow has the following stages:
+1. **Quota reservation**: quota is reserved, and the workload obtains the
+  `QuotaReserved` condtion. Preemptions are evaluated if configured.
+2. **Admission checks**: waiting for all AdmissionChecks, including the
+  Provisioning one, to report `Ready`.
```

Comment:
```suggestion
  Provisioning one. Once all AdmissionChecks are ready, the Workload gets `Ready` admissionCheck state.
```

Does this `Ready` indicate for `AdmissionCheck` or `Workload`?

Reply:
it is under `workload.status.admissionChecks[].state`

Reply:
>  Once all AdmissionChecks are ready, the Workload gets `Ready` admissionCheck state.

This is not exactly accurate I think, because the fields `workload.status.admissionChecks[].state` are updated by AC controllers one by one

Reply:
```
2. **Admission checks**: The Workloads wait for all AdmissionChecks Ready, including the
  Provisioning one.
```

In that case, how about this one?